docs/user_guide/configuration/slackv3.rst

```diff
@@ -22,15 +22,18 @@ installed in a python virtual environment (adjust the command to your errbot's i
     /opt/errbot/bin/pip install -r /opt/errbot/backends/err-backend-slackv3/requirements.txt
```

Comment:
👋 @nzlosh since you are working on this documentation PR, I found that `requirements.txt` has been removed from the repo in this PR 
https://github.com/errbotio/err-backend-slackv3/pull/82/files#diff-4d7c51b1efe9043e44439a949dfd92e5827321b34082903477fd04876edb7552.